.github/workflows/test.yml

```diff
@@ -51,3 +51,17 @@ jobs:
             pip install tox tox-gh-actions
         - name: Run integration tests
           run: tox -eintegration
+    twine:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v2
+        - name: Setup Python
+          uses: actions/setup-python@v2
+          with:
+            python-version: 3.9
+        - name: Install Tox and any other packages
+          run: |
+            python -m pip install --upgrade pip
+            pip install tox tox-gh-actions
+        - name: Run twine-check
+          run: tox -etwine
```

Comment:
It looks like the little warning sign here means there's no newline at the end of the file, you may want to add them to the affected files.